staging/src/k8s.io/apiserver/pkg/endpoints/handlers/get.go

```diff
@@ -307,3 +312,19 @@ func ListResource(r rest.Lister, rw rest.Watcher, scope *RequestScope, forceWatc
 		transformResponseObject(ctx, scope, req, w, http.StatusOK, outputMediaType, result)
 	}
 }
+
+type watchListEndpointRestrictions struct {
+	negotiation.EndpointRestrictions
+}
+
+func (e *watchListEndpointRestrictions) AllowsMediaTypeTransform(mimeType, mimeSubType string, target *schema.GroupVersionKind) bool {
+	if target != nil && target.Kind == "Table" {
+		return false
+	}
+	return e.EndpointRestrictions.AllowsMediaTypeTransform(mimeType, mimeSubType, target)
+}
+
+// TODO: check ProgressNotification ?
```

Comment:
This should be caught elsewhere already, no?

Reply:
in https://github.com/kubernetes/kubernetes/pull/127587